### ISSUE TITLE
Change color of output?

### ISSUE THREAD

**Issue by theonlygusti** [wting/autojump#489]
Can I change the color to something other than the blaring red? It doesn't work so well with my current color scheme.

**Comment by Hund** [comment]
I would like to know this as well.

**Comment by Hund** [comment]
This is possible to solve yourself, open the file your shell sources, in Arch it's the file `/etc/profile.d/autojump.zsh` and find this line:

`echo -e "\\033[31m${output}\\033[0m"`

Change the value `31` to whatever you like and reload zsh. :)

**Comment by wting** [comment]
Yeah I should probably just remove the color or even the output altogether.

**Comment by la0s** [comment]
For the brew installation,I use commend grep -rn "color" /usr/local/Cellar/autojump find location and change /usr/local/Cellar/autojump/22.5.1/share/autojump/autojump.zsh
